packages/eslint-rules/lib/rules/no-custom-color.js

```diff
@@ -0,0 +1,174 @@
+const tailwindConfig = require("../../../shared/tailwind.config");
+const alwaysValidClasses = [
+  "text-base",
+  "text-center",
+  "text-2xl",
+  "text-left",
+  "text-lg",
+  "text-xl",
+  "bg-none",
+];
+
+module.exports = {
+  meta: {
+    type: "problem",
+    docs: {
+      description: "Incorrect command class name",
+      suggestion: false,
+    },
+    fixable: "code",
+  },
+  create: (context) => {
+    const targetClassNames = ["bg-", "text-"];
```

Comment:
What about the border and shadow props?

Reply:
We can add those, but I think the default tailwind plugin actually already checks for those.